### PR TITLE
Lit6498 astawadeo

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
It seems that I intended to create a work record for a unique Amharic work preserved in BLorient792.

### CODE REVIEW

**Comment by eu-genia** [comment]
Since @CarstenHoffmannMarburg had several mss with Amharic commentaries maybe he can check this one too, thank you!

File: new/LIT6498Astawadeo.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The work contains mostly histories of the Old Testament personages and commentaries to passages from the liturgical texts.</p>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6498Astawadeo.xml:34]: I suggest: "The work contains mostly stories of Old Testament personages (or: "characters") and commentaries on passages of liturgical texts."

Comment (CarstenHoffmannMarburg) [reply new/LIT6498Astawadeo.xml:34]: You could also add "in Amharic" to make it clear that the main language is Amharic.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
Thank you! Very nice job!

**Review by eu-genia — APPROVED** [review]

**Comment by MarcinKrawczuk** [comment]
Thanks Carsten for the suggestions, done. I'm merging.

**Pull request merged**